### ISSUE TITLE
Train model in PyTorch for Leafs dataset

### ISSUE THREAD

**Issue by cwmeijer** [dianna-ai/dianna-exploration#58]
Blocked by #44.
Script to train the model should be stored in repo
Resulting model (ONNX) saved in the surfdrive

**Comment by loostrum** [comment]
The dataset had to be reorganized to be able to read it easily with torchvision: there should be one folder, with subfolders per class, that each contain all images of that class.
We should either apply the same reorganization to the dataset on Surfdrive, or define a custom dataloader.

**Comment by loostrum** [comment]
Updated dataset with required subfolder structure was added to surfdrive. It's also split in a train/test/validation set, with the same division in field/lab images as the full dataset. 

After hyperparameter sweep with W&B, the best model was converted to ONNX and stored to surfdrive (accuracy on validation set: 86%, test set: 84%)

**Comment by loostrum** [comment]
Resulting confusion matrix: 
![leafsnap_confusion_matrix](https://user-images.githubusercontent.com/6370787/121338792-24e32580-c91e-11eb-827d-24505250a768.png)
